routes/index.js

```diff
@@ -76,6 +76,10 @@ var routes = {
     };
     var request_id = request.headers['x-request-id'];
 
+    if (transaction.description === "Thunderbird") {
+      metadata.thunderbird = true;
+    }
```

Comment:
Is there a reason you need it as a flag and not a desc? Or you just feel this is better overall?